env/environment.yml

```
@@ -0,0 +1,26 @@
+name: isce3-ci-sandbox
+channels:
+  - conda-forge
+  - nodefaults
+dependencies:
+  - backoff
+  - cmake>=3.18
+  - cxx-compiler
+  - eigen>=3.3
+  - fftw>=3.3
+  - gdal>=3.0
+  - gmock>=1.10
+  - gtest>=1.10
+  - h5py>=3.0
+  - hdf5>=1.10.2,<1.13
```

Initially I set this upper bound on the HDF5 version, because my experience is that the HDF5 folks don't follow [semantic versioning](https://semver.org/) and tend to introduce a lot of breaking changes even in their patch releases. So when I'm teaching people to build isce3, I try to avoid building with a newer version HDF5 than I personally have tested with, in case some new breaking changes have been introduced.

But it occurs to me now that we should remove this upper bound for purposes of CI testing for basically the same reason -- if a new breaking change is introduced in a recent HDF5 version, it would be good for the CI system to let us know early on so we can address it quickly.

```suggestion
  - hdf5>=1.10.2
```